Source/Core/Core/FifoPlayer/FifoRecordAnalyzer.cpp

```diff
@@ -227,78 +228,3 @@ void FifoRecordAnalyzer::WriteVertexArray(int arrayIndex, u8 *vertexData, int ve
 
 	FifoRecorder::GetInstance().WriteMemory(arrayStart, arraySize, MemoryUpdate::VERTEX_STREAM);
 }
-
-void FifoRecordAnalyzer::ProcessTexMaps()
```

Comment:
"ProcessLoadIndexedXf" and "ProcessVertexArrays" are still missing. To be honest, they are the easier ones :D